tests/expectations/core/test_expect_queried_table_row_count_to_be.py

```diff
@@ -126,10 +126,9 @@ def test_expect_queried_column_value_frequency_to_meet_threshold_spark(
     row_condition,
     spark_session,
     basic_spark_df_execution_engine,
+    titanic_df,
 ):
-    df = pd.read_csv(
-        "tests/test_fixtures/configuration_for_testing_v2_v3_migration/data/Titanic.csv"
-    )
+    df = titanic_df
```

Comment:
@Shinnnyshinshin Same: requesting type hints.  Thank you!

Reply:
done